hack/prometheus-deploy.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+#
+# This file is part of the Kepler project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2024 The Kepler Contributors
+#
+
+set -eu -o pipefail
+
+PROJECT_ROOT="$(git rev-parse --show-toplevel)"
+declare -r PROJECT_ROOT
+
+source "$PROJECT_ROOT/hack/utils.bash"
+
+declare PROM_DIR="kube-prometheus"
+declare KEPLER_EXPORTER_GRAFANA_DASHBOARD_JSON=$( curl -fsSL https://raw.githubusercontent.com/sustainable-computing-io/kepler/main/grafana-dashboards/Kepler-Exporter.json | sed '1 ! s/^/         /' )
+
+clone_prometheus(){
+    if [ -d "$PROJECT_ROOT/$PROM_DIR" ]; then
+        return 0
+    fi
+    header "Clone kube-prometheus"
+    run git clone --depth 1 https://github.com/prometheus-operator/kube-prometheus
```

Comment:
https://github.com/sustainable-computing-io/local-dev-cluster/blob/7dafcdaeec51e781bb771ad75140ef864029b758/lib/prometheus.sh#L36
please update and enhance on current scripts.